packages/headless/src/features/insight-search/insight-case-context-state.ts

```diff
@@ -0,0 +1,8 @@
+export interface InsightCaseContextState {
+  caseContext: Record<string, string>;
+}
+
+export const getInsightCaseContextSearchInitialState =
```

Comment:
Have you thought about how/where we should update this case context state? It looks like that redux mechanism isn't implemented here so we will need a separate story for that.
My hunch is it'll be preferable to have it in its own features/insight-case-context folder but I'd like to know your idea for this.

Reply:
I think about this too, I think it would be better another PR to create action that set `CaseContext`

Reply:
Could you create a jira for that please?

Reply:
https://coveord.atlassian.net/browse/SVCC-2249